tsconfig.base.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
Ok, now I see why @szmarczak is talking about "a lot of config files". Can we make this better somehow? This config will get out of sync between packages sooner or later.